src/boards/include/boards/pimoroni_picosystem.h

```diff
@@ -115,15 +117,39 @@
 #define PICO_DEFAULT_UART_RX_PIN 1
 #endif
 
+// --- LED ---
 // Included so basic examples will work, and set it to the green LED
 #ifndef PICO_DEFAULT_LED_PIN
 #define PICO_DEFAULT_LED_PIN PICOSYSTEM_LED_G_PIN
 #endif
+// no PICO_DEFAULT_WS2812_PIN
 
 #ifndef PICO_DEFAULT_LED_PIN_INVERTED
 #define PICO_DEFAULT_LED_PIN_INVERTED 1
 #endif
 
+// --- I2C ---
+//no PICO_DEFAULT_I2C
+//no PICO_DEFAULT_I2C_SDA_PIN
+//no PICO_DEFAULT_I2C_SCL_PIN
+
+// --- SPI ---
+#ifndef PICO_DEFAULT_SPI
+#define PICO_DEFAULT_SPI 0
+#endif
+#ifndef PICO_DEFAULT_SPI_SCK_PIN
+#define PICO_DEFAULT_SPI_SCK_PIN PICOSYSTEM_LCD_SCLK_PIN
```

Comment:
Does it make sense to define these as `PICO_DEFAULT_SPI_SCK_PIN` if they're hardwired to the LCD?

Reply:
You are right. In this case the SPI is not exposed onto any headers, so the specific LCD defines should be used. I have added a `PICOSYSTEM_LCD_SPI` define in place of `PICO_DEFAULT_SPI`